src/commands/ssh-keygen.ts

```diff
@@ -0,0 +1,22 @@
+/** Copyright © 2024-present P0 Security
+
+This file is part of @p0security/cli
+
+@p0security/cli is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3 of the License.
+
+@p0security/cli is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
+**/
+import { createKeyPair, P0_KEY_FOLDER } from "../common/keys";
+import yargs from "yargs";
+
+export const sshKeyGenCommand = (yargs: yargs.Argv) =>
+  yargs.command(
+    "ssh-keygen",
+    `Generate an SSH key pair for use with P0 and store it in your ${P0_KEY_FOLDER} folder, if one does not exist already. This key pair is also automatically generated by using any "p0 ssh/scp ..." command.`,
+    (yargs) => yargs,
+    async () => {
+      await createKeyPair();
+    }
```

Comment:
nit:
```suggestion
    createKeyPair
```

Reply:
`createKeyPair` has a return value so we have to do this